python/hsfs/feature_group.py

```diff
@@ -1675,6 +1698,151 @@ def insert(
             ge_report.to_ge_type() if ge_report is not None else None,
         )
 
+    def multi_part_insert(
+        self,
+        features: Union[
+            pd.DataFrame,
+            TypeVar("pyspark.sql.DataFrame"),  # noqa: F821
+            TypeVar("pyspark.RDD"),  # noqa: F821
+            np.ndarray,
+            List[list],
+        ] = None,
+        overwrite: Optional[bool] = False,
+        operation: Optional[str] = "upsert",
+        storage: Optional[str] = None,
+        write_options: Optional[Dict[str, Any]] = {},
+        validation_options: Optional[Dict[str, Any]] = {},
+    ) -> Tuple[Optional[Job], Optional[ValidationReport]]:
```

Comment:
Shouldn't this include the writer as well?

Reply:
true, done